### ISSUE TITLE
Auto Share

### ISSUE THREAD

**Issue by CaiusTSM** [Fankserver/atom-firepad#21]
Would it be possible to have an option to automatically share every file in the current working directory?

**Comment by Fank** [comment]
yea but how should it be handled?
a config file in the project folder, where you can setup the which files should be shared automaticly and which share key should be used?

**Comment by CaiusTSM** [comment]
Yes I believe a file that holds the share key(s) would be required. Something along the lines of file_path (relative to project) and the associated share key (I do not know much about how firebase/firepad works, but I am assuming it is one share key per file). Perhaps also something like a .gitignore (.afignore) so it does not share everything as that would cause problems when the project becomes very big and has some files/folders that you do not want to share (such as node_modules). So instead of picking which files to share, you pick which files/folders not to share.
